CODEOWNERS

```diff
@@ -70,3 +70,8 @@
 /post-processor/yandex-export/          @GennadySpb
 /post-processor/yandex-import/          @GennadySpb
 /post-processor/vsphere-template/       nelson@bennu.cl
+
+# release configuration
```

Comment:
I pushed an update to fix the issue. This may cause a conflict here that needs to be pulled in.

Reply:
This is great. TY!